orca-core/orca-core.gradle

```diff
@@ -49,6 +49,7 @@ dependencies {
   implementation("org.apache.commons:commons-lang3")
   implementation("javax.servlet:javax.servlet-api:4.0.1")
   implementation("com.jayway.jsonpath:json-path:2.2.0")
+  implementation("org.yaml:snakeyaml")
```

Comment:
where does the version for `snakyaml` come from?

Reply:
from kork? I'm not very familiar with how the new dependency definition works :/ I saw there are other dependencies without version, just thought it would be ok not to pin.

Reply:
Apologies for the delay -  was out of town.
Interesting, I don't see it specified in `kork`...  somehow it gets into kork-bom. oh well

Reply:
looks like it comes from `spring-boot-dependencies` (pulled in by `kork`: https://search.maven.org/artifact/org.springframework.boot/spring-boot-dependencies/2.1.4.RELEASE/pom)

Reply:
thanks for the pointer 👍